packages/cli/src/lib/bundler/config.ts

```diff
@@ -77,16 +79,65 @@ export function createConfig(
     }),
   );
 
+  const backendRelatedConfig = {
+    watch: true,
+    watchOptions: {
```

Comment:
I think this config should be added in as a part of setting up watch mode instead. Basically splitting it in the same way we'd do if we also want to do a non-watch build.